### PR TITLE
As an admin user I want to be able to see a list of questions

### PR DESCRIPTION
**Pull request by OrestF**
Add controller questions_controller with index method.
Create user, templates, topics, questions models.

### CODE REVIEW

**Comment by freiwillen** [comment]
.idea/Aid-Interview.iml - must be in gitignore because this related only to your workspace.

db/migrate/20151126150623_create_users.rb, db/migrate/20151126150811_create_topics.rb, db/migrate/20151126150844_create_templates.rb, app/models/user.rb, app/models/topic.rb, app/models/template.rb are not related to this task and related to others, so should not be in this PR

**Comment by serhiym** [comment]
@OrestF - add next line to gitignore to avoid idea files to be added
```
/.idea
```
also try using 'git add -p' command instead of 'git add .'
in this case you'll see what you're adding to commit

File: Gemfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ group :test do
 end
 
 group :production do
-  gem 'pg',             '0.17.1'
+  gem 'mysql2'
```
Comment (serhiym) [inline Gemfile]: if we 'll use heroku as production
then we should leave pg gem
@freiwillen - please confirm

**Comment by serhiym** [comment]
@OrestF it's a good idea to add pagination to questions list
please add (you already did it in rails tutorial)

File: Gemfile (hunk 1 above)
Comment (freiwillen) [inline Gemfile]: ^^ :+1:

**Pull request closed without merging**